trymito.io/excel-to-python-page-contents/CONCAT.json

```diff
@@ -0,0 +1,121 @@
+{
+    "slug": ["functions", "text", "CONCAT"],
+    "functionNameShort": "CONCAT",
+    "functionNameLong": "String Concatenation",
+    "relatedFunctions": ["JOIN", "MERGE", "APPEND"],
+    "titleCardParagraphs": [
+        "Excel's CONCAT function combines two or more strings into one. It's a powerful tool in text-based data analysis, and is often used to combine fields or create unique identifiers.",
+        "This page explains how to implement Excel's CONCAT function in Python using pandas."
+    ],
+    "excelExplanation": {
+        "paragraphs": [
+            "The CONCAT function in Excel takes two or more string arguments and combines them into one continuous string.",
+            "=CONCAT(text1, [text2], ...)"
+        ],
+        "syntaxTable": [
+            {
+                "parameter": "text1",
+                "description": "The first text value you want to concatenate.",
+                "dataType": "string"
+            },
+            {
+                "parameter": "text2",
+                "description": "(Optional) Additional text values to concatenate.",
+                "dataType": "string"
+            }
```

Comment:
I think we could use some way to represent the ..., aka a variable number of arguments after this.